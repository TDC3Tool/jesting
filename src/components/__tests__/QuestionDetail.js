describe(`Question Detail component`, () => {
    it(`should not regress`, () => {


    });
});