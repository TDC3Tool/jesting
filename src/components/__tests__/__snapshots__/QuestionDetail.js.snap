// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` 1`] = `
<div>
  <h3
    className="mb-2"
  >
    The meaning of life
  </h3>
  <div>
    <div
      className="mb-3"
    >
      <div>
        <code>
          hitchhiking
        </code>
      </div>
    </div>
    <div>
      <p>
        42
      </p>
    </div>
    <div>
      10
       Answers
    </div>
  </div>
</div>
`;
